saleor/setting/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy
+from django.utils.encoding import python_2_unicode_compatible
+
+
+INTEGER = 'i'
+STRING = 's'
+BOOLEAN = 'b'
+
+
+@python_2_unicode_compatible
+class Setting(models.Model):
```

Comment:
We can't use the `Site` model as it's cached per process so changes are only visible after the server is restarted. We could however benefit from a similar object being present (there are places that need to know the current domain, site name etc.).

Maybe we should fork `django.contrib.sites` as an external repository and add support for storing settings there (again, I think we would be fine with settings stored like we store checkout data)?